docs/resources/project_ssh_key.md

```diff
@@ -26,8 +26,8 @@ resource "metal_project_ssh_key" "test" {
 resource "metal_device" "test" {
   hostname            = "test"
   plan                = "baremetal_0"
```

Comment:
`git grep plan.*[12_] metal/ docs/ | grep -v _test.go` there are a few legacy plans captured in this grep.

Reply:
thx will fix later today